### ISSUE TITLE
rnpm will be deprecated in a future release of React Native

### ISSUE THREAD

**Issue by tylerolechnowicz** [react-native-community/react-native-maps#3181]
# Bug

<!--
  Please provide a clear and concise description of what the bug is.
  Include screenshots if needed.
  Please test using the latest release of the library, as maybe said bug has been already fixed.
  If the library has multiple install methods, describe installation method (e.g., pod, not pod, with jetifier etc)
-->

Getting this warning when running my project: 
![image](https://user-images.githubusercontent.com/32344917/68350325-f19b3d80-00cd-11ea-8f40-2a1a352021f5.png)


## Environment info

<!--
  Run `react-native info` in your terminal and copy the results here. Also, include the *precise* version number of this library that you are using in the project
-->

React native info output:

```bash
warn The following packages use deprecated "rnpm" config that will stop working from next release:                                                                                                                  - react-native-maps: https://github.com/react-native-community/react-native-maps#readme                                                                                                                         Please notify their maintainers about it. You can find more details at https://github.com/react-native-community/cli/blob/master/docs/configuration.md#migration-guide.                                           info Fetching system and libraries information...                                                                                                                                                                 System:                                                                                                                                                                                                               OS: Windows 10                                                                                                                                                                                                    CPU: (8) x64 Intel(R) Core(TM) i7-6820HK CPU @ 2.70GHz                                                                                                                                                            Memory: 7.30 GB / 31.87 GB                                                                                                                                                                                      Binaries:                                                                                                                                                                                                           Node: 10.15.1 - C:\Program Files\nodejs\node.EXE                                                                                                                                                                  Yarn: 1.15.2 - ~\AppData\Roaming\npm\yarn.CMD                                                                                                                                                                     npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD                                                                                                                                                                    SDKs:                                                                                                                                                                                                               Android SDK:                                                                                                                                                                                                        API Levels: 28, 29                                                                                                                                                                                                Build Tools: 28.0.3, 29.0.2                                                                                                                                                                                       System Images: android-27 \| Google APIs Intel x86 Atom, android-28 \| Intel x86 Atom_64, android-28 \| Google APIs Intel x86 Atom_64, android-28 \| Google Play Intel x86 Atom                                   IDEs:                                                                                                                                                                                                               Android Studio: Version  3.5.0.0 AI-191.8026.42.35.5900203                                                                                                                                                      npmPackages:                                                                                                                                                                                                        react: ^16.11.0 => 16.11.0                                                                                                                                                                                        react-native: ^0.60.6 => 0.60.6
```

Library version: ^0.25.0

**Comment by eggsys** [comment]
What does rnpm do? is that a part of config 
i can't find document about rnpm.
i'm newbie for RN

**Comment by yagoag** [comment]
This warning should not prevent you from using the library, but I would advise you to update to the latest version.
This has already been solved in version 0.26.0 (see #2975)

**Comment by rborn** [comment]
Closing as invalid (with the latest version)
